solutions/problem3.js

```diff
@@ -0,0 +1,9 @@
+const { isArrayEmpty } = require('./utils')
+
+const first = (promises, errors = []) => isArrayEmpty(promises) 
```

Comment:
To nasza wina, gdyż źle napisaliśmy testy, ale to nie jest niestety dobre rozwiązanie. Domerdżuj sobie mastera, gdzie zrobiliśmy update testów i zastanów się, czemu testy nie przechodzą.